containers.go

```diff
@@ -84,7 +84,7 @@ type ContainerStore interface {
 
 	// SetNames updates the list of names associated with the container
 	// with the specified ID.
-	SetNames(id string, names []string) error
+	SetNames(id string, names []string, opts SetNameOptions) error
```

Comment:
Still a breaking change.

Reply:
@vrothberg Yeah i was not aware that container store is exposed dropped it.